app/components/common/root.js

```diff
@@ -14,6 +15,7 @@ class Root extends React.Component {
     await this.props.fetchUser();
     await this.props.fetchOrders();
     await this.props.fetchCart();
+    await this.props.fetchReviews();
```

Comment:
We don't need to fetch all the reviews - any reviews we need access to will be able to be accessed via the associated product